src/app/components/login/login.jsx

```diff
@@ -154,6 +154,15 @@ function ChplLogin(props) {
     }
   };
 
+  const toastWhenUsernameUsed = (enteredUsername, loggedInUser) => {
+    if (enteredUsername !== loggedInUser.email) {
+      toaster.pop({
+        type: 'Warn',
```

Comment:
Our other toasters have a header. I think we should keep that consistent

```suggestion
        header: 'Warning',
```